observability-lib/grafana/panels.go

```diff
@@ -453,20 +483,45 @@ func NewTablePanel(options *TablePanelOptions) *Panel {
 
 type LogPanelOptions struct {
 	*PanelOptions
-	PrettifyJSON bool
+	ShowTime         bool
+	PrettifyJSON     bool
+	EnableLogDetails *bool
+	DedupStrategy    common.LogsDedupStrategy
+	SortOrder        common.LogsSortOrder
 }
 
 func NewLogPanel(options *LogPanelOptions) *Panel {
 	setDefaults(options.PanelOptions)
 
+	if options.EnableLogDetails == nil {
+		options.EnableLogDetails = Pointer[bool](true)
+	}
```

Comment:
So we enable by default log details

Reply:
Yes, most of the defaults that I have added are the same defaults if you were to use the UI.